website/docs/Migration20.md

```diff
@@ -39,9 +39,29 @@ Both changes should improve code readibility.
 Please note that in many cases `waitFor` call can be replaced by proper use of `findBy` asynchonous queries resulting in more streamlined test code.
 :::
 
-## Removed global `debug` function
+## Removed global debug functions
 
-Global debug function has been removed in favor of `debug()` method returned from `render()` function.
+Global `debug()` and `shallow()` functions have been removed in favor of `debug()` method returned from `render()` function.
+
+## Removed functions
+
+Following query fuctions have been removed after being deprecated:
+
+- `getByName`
+- `getAllByName`
+- `queryByName`
+- `queryAllByName`
+
+Following query functions are still available but have been prefixed with `UNSAFE_`:
+
+- `UNSAFE_getByType`
+- `UNSAFE_getAllByType`
+- `UNSAFE_queryByType`
+- `UNSAFE_queryAllByType`
+- `UNSAFE_getByProps`
+- `UNSAFE_getAllByProps`
+- `UNSAFE_queryByProps`
+- `UNSAFE_queryAllByProps`
```

Comment:
Maybe something like this would be more descriptive?
```
The `*ByType` and `*ByProps` queries has been prefixed with `UNSAFE_`. You can safely rename them using global search/replace in your project:
- `getByType` ->`UNSAFE_getByType`
- ...
```

Reply:
Done